src/toil/fileStore.py

```diff
@@ -1603,7 +1607,8 @@ def open(self, job):
                                            disk=diskUsed,
                                            humanRequestedDisk=bytes2human(jobReqs),
                                            requestedDisk=jobReqs))
-            self.logToMaster(logString, level=logging.INFO)
+            self.logToMaster(logString, level=(logging.DEBUG if self.jobName.startswith(self.debugJobNames)
```

Comment:
I don't think this logic is necessary. We can just make this statement DEBUG outright since the next log message is level WARNING but only logs when the job uses more disk than requested.